libs/execution_context/ebpf_core.c

```diff
@@ -899,6 +933,24 @@ _ebpf_core_map_delete_element(ebpf_map_t* map, const uint8_t* key)
     ebpf_map_delete_entry(map, key);
 }
 
+static long
+_ebpf_core_tail_call(void* context, ebpf_map_t* map, uint32_t index)
```

Comment:
should there be some kind of check somewhere that the map type is PROGRAM_ARRAY?

Reply:
Good point, will fix

Reply:
done